libraries/chain/hardfork.d/CORE_1268.hf

```diff
@@ -0,0 +1,4 @@
+// #1268 Distribute Asset Market Fees to Referral Program 
+#ifndef HARDFORK_1268_TIME
+#define HARDFORK_1268_TIME (fc::time_point_sec( 1530705600 )) // Wednesday, July 4, 2018 12:00:00 PM
```

Comment:
Of course, we will change it during the last iteration. Right now the current value is more suitable for us because our smoke tests cover the market fee sharing logic after HARDFORK_1268_TIME

Reply:
Done.